addons/html_editor/static/src/editor/link/link_dialog.js

```diff
@@ -0,0 +1,192 @@
+/** @odoo-module **/
+
+import { _t } from "@web/core/l10n/translation";
+import { useService, useChildRef } from "@web/core/utils/hooks";
+import { Dialog } from "@web/core/dialog/dialog";
+import { Link } from "./link";
+
+export class LinkDialog extends Link {
```

Comment:
using inheritance is weird, but i may not understand the usecase very well. usually, in owl, we want to use composition (sub components) instead of inheritance